tests/test_user.py

```diff
@@ -427,15 +424,15 @@ def test_account_creation_by_staff(self):
             'organizations': [{'organization_id': org_id},
                               {'organization_id': org2_id}],
             'consents': [{'organization_id': org_id,
-                        'agreement_url': 'http://fake.org',
-                        'staff_editable': True,
-                        'send_reminders': False}],
+                          'agreement_url': 'http://fake.org',
```

Comment:
Too much indentation, are you trying to align it with the above line?

Reply:
please don't indent more than one level

Reply:
Hmm still seeing it in the [diff](https://github.com/uwcirg/true_nth_usa_portal/pull/2515/files), give it another try?

Reply:
@ivan-c still see it?